pkg/agent/controller/traceflow/packetin.go

```diff
@@ -250,14 +249,16 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1alpha1.Tracefl
 		if pktIn.Data.Ethertype == protocol.IPv6_MSG {
 			gatewayIP = c.nodeConfig.GatewayConfig.IPv6
 		}
-		if c.networkConfig.TrafficEncapMode.SupportsEncap() && outputPort == config.DefaultTunOFPort {
+		gwPort := c.nodeConfig.GatewayConfig.OFPort
+		tunPort := c.nodeConfig.TunnelOFPort
```

Comment:
Why we use nodeConfig in Traceflow but use ifaceStore in NetworkPolicy to get gwPort and tunPort?

Reply:
The ofport value is stored in both interfaceStore and NodeConfig. And it is easier to get it in NodeConfig. NetworkPolicy Controller does not have a property for NodeConfig that's why I have to use interfaceStore.

Reply:
Could  you add comment to explain why NetworkPolicy uses a "not recommended" method?